data/operators/amenity/charging_station.json

```
@@ -1743,22 +1725,23 @@
     {
       "displayName": "ESB ecars",
       "id": "esbecars-179e3e",
-      "locationSet": {"include": ["ie"]},
-      "matchNames": ["esb"],
+      "locationSet": {"include": ["ie", "gb-nir"]},
+      "matchNames": ["esb", "ecarni"],
       "tags": {
         "amenity": "charging_station",
-        "operator": "ESB ecars",
-        "operator:wikidata": "Q3050566"
+        "operator": "ESB Group",
```

I have no knowledge of this brand, but should this be?

```suggestion
        "operator": "ESB",
```

> Charge your electric vehicle on the ESB public charging network

> ESB owns, operates and maintains over 1,350 public charge points across the island of Ireland. Check out our interactive map to find the nearest ESB charge point to you

https://esb.ie/what-we-do/ecars

---

Oh, I've just seen a `brand` as well.

I've got no strong opinion on this.